objects/objects.shacl.ttl

```diff
@@ -489,6 +490,24 @@
       sh:message "ebucore:isMediaFragmentOf is geen premis:File"@nl ; 
       sh:message "ebucore:isMediaFragmentOf n'est pas un premis:File"@fr ;
       sh:severity sh:Violation ;
+    ],
+    [
+      sh:path  premis:identifier ;
```

Comment:
doortrekken naar alle representations?

Reply:
Je bedoelt of we alle MH IDs op representatieniveau kunnen definiëren? Ik denk dat die doorgaans eerder filegebonden zijn, buiten dus de fragment identifier die ik hier zou definiëren om te vermijden dat een file waar meerdere fragmenten van aangemaakt worden, allemaal fragment identifiers krijgt toegewezen.